### DISCUSSION TITLE
Parking

### DISCUSSION THREAD

**Discussion by Klippert** [terjeio/grblHAL#292] [Q&A]
Hello,

I find parking very useful with the door trigger. Is there a way to go to a "end-of-job-parking" position so i don't have to jog the machine out of the way everytime after a completed job?

**Comment by terjeio** [comment]
Yes, use [G28](http://www.linuxcnc.org/docs/2.6/html/gcode/gcode.html#sec:G28-G28_1) or G30 to set the position and move there.

**Reply by Klippert** [reply]
Would you then add a Gcode to the postprocessor to add a move command at the end of each job? 
or would you just manually do a move at the end of the job?

**Reply by terjeio** [reply]
Your choice. If using ioSender then buttons in the right hand _Goto_ flyout can be used for the move.

**Reply by Klippert** [reply]
aha! - ok. i was thinking of some way of automating it. but it's pretty easy that way too :)
